libraries-rpc/src/main/java/com/baeldung/rpc/finagle/LogFilter.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.rpc.finagle;
+
+import com.twitter.finagle.Service;
+import com.twitter.finagle.SimpleFilter;
+import com.twitter.finagle.http.Request;
+import com.twitter.finagle.http.Response;
+import com.twitter.util.Future;
+
+public class LogFilter extends SimpleFilter<Request, Response> {
+    @Override
+    public Future<Response> apply(Request request, Service<Request, Response> service) {
+        System.out.println("Request host:" + request.host().getOrElse(() -> ""));
+        System.out.println("Request params:");
```

Comment:
if the stdouts are irrelevant let's remove them

Reply:
The whole point of this class is to log things so I wouldn't say they are irrelevant but maybe there is some standard logger library to use instead. If so, let me know, I will replace stdouts.

Reply:
since you are importing the logback already, then I would say we use the logging facility

Reply:
Ok, I replaced stdouts with a logger.